gradle/libs.versions.toml

```diff
@@ -61,12 +61,12 @@ hadoop3           = "3.3.6"
 # when updating this version, you need to ensure compatibility with:
 #  - plugins/ingest-attachment (transitive dependency, check the upstream POM)
 #  - distribution/tools/plugin-cli
-bouncycastle_jce  = "2.0.0"
-bouncycastle_tls  = "2.0.19"
-bouncycastle_pkix = "2.0.7"
-bouncycastle_pg   = "2.0.10"
-bouncycastle_util = "2.0.3"
-password4j        = "1.8.2"
```

Comment:
Why remove password4j? is this a typo?

Reply:
Password4j is used only in the security plugin, not in the core.

Reply:
Ahh my bad it is in the core